### ISSUE TITLE
Guardian Node Synchronisation Issue

### ISSUE THREAD

**Issue by ThetaTokenNetworkGithub** [thetatoken/guardian-mainnet-guide#170]
Hi @greer3, Has your guardian node synchronisation issue been solved ? If not I'll be opening a new ticket ID for you

**Comment by greer3** [comment]
No.  The 10,000 tfuel has not shown up in the counter window that tracks what’s been stake.
Thank you
Greer
